changelog.md

```diff
@@ -1,4 +1,7 @@
 # Future
+- [Fixed] `findAndCountAll` generates wrong SQL query [#5445](https://github.com/sequelize/sequelize/issues/5445)
+- [Fixed] `count` methods pollute the options.includes [#4191](https://github.com/sequelize/sequelize/issues/4191)
```

Comment:
#4191 , I have added a test for that https://github.com/sushantdhiman/sequelize/commit/3f15e064f274c2219c982f4b7fffe757956f6ac0